airlock/settings.py

```diff
@@ -101,14 +102,14 @@ def get_env_var(name):
     "django.contrib.sessions.middleware.SessionMiddleware",
     "django.middleware.common.CommonMiddleware",
     "django.middleware.csrf.CsrfViewMiddleware",
-    # "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    # "django.contrib.auth.middleware.LoginRequiredMiddleware",
```

Comment:
Should we just remove the django LoginRequiredMiddleware? We're not planning to use it in future are we?

Reply:
Yeah, I though we might do.

I will remove in the next PR though, to avoid changing this and having to re-approve, if that's ok